qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -554,11 +554,11 @@ def run(self, run_input, **kwargs):
             )
         if pulse_job:
             if _optionals.HAS_AER:
-                from qiskit.providers import aer
-                from qiskit.providers.aer.pulse import PulseSystemModel
+                import qiskit_aer
+                from qiskit_aer.pulse import PulseSystemModel
```

Comment:
Note https://github.com/Qiskit/qiskit/pull/11442#discussion_r1433917371

Reply:
Added a deprecation warning in b299aae.